### DISCUSSION TITLE
cy.login() is not working

### DISCUSSION THREAD

**Discussion by Aliwahid17** [prescottprue/cypress-firebase#718] [Q&A]
### Error 

defaultApp.auth is not a function

`
48 | function getAuth(tenantId) {

49 | const auth = defaultApp.auth() ;
| ^
50 | if (tenantId) {
51 | auth.tenantId = tenantId;
52 | }

`

**Comment by prescottprue** [comment]
Sounds like you may have missed part of #2 in [the setup docs](https://github.com/reside-eng/admin-ui/blob/cypress-tools/.github/workflows/merge-main-base-build.yml#L38) which includes initializing the Firebase JS sdk - without doing that there is no default app to use.
